src/main/resources/site/site.xml

```diff
@@ -1,109 +1,134 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <site>
     <form>
-        <option-set name="contactOptions">
-            <label i18n="contact">Contact</label>
+        <option-set name="options">
+            <label i18n="options.label">Use textarea or inputfields</label>
```

Comment:
pirk, men "input fields" skrives i to ord, vanligvis. Det er noen tilfeller lenger opp også.